pages/alephium-web3-providers.tsx

```diff
@@ -235,21 +235,20 @@ const AlephiumWeb3Provider = ({ children }: AlephiumWeb3ProviderProps) => {
         })
 
         const windowAlephium = await connect({ showList: false })
-
         if (windowAlephium) {
           await windowAlephium.enable()
-          const accounts = await windowAlephium.getAccounts()
+          const selectedAccount = await windowAlephium.getSelectedAccount()
 
           windowAlephium.on("addressesChanged", (_data) => {
             dispatch({
               type: 'SET_SELECTED_ACCOUNT',
-              selectedAccount: windowAlephium.selectedAccount
+              selectedAccount: selectedAccount
```

Comment:
Question: do we need to call `windowAlephium.getSelectedAccount()` when address changed?